proximal/halide/src/test_data/meson.build

```diff
@@ -0,0 +1,4 @@
+# TODO(Antony): Request Mesonbuild for the API: files(...).get_path()
+#parrot_img = files('parrot.png')
```

Comment:
can you clean up this comment?

Reply:
Sure. I un-commented the line. It asserts that the test image exists in the C++ project folder.